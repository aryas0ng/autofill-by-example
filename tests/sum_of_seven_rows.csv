11,42
15,74
2,64
5,68
3,72
2,
4,
43,
5,
6,
9,
3,
4,
2,
5,
2,
6,
1,
8,
6,
1,
21,
3,