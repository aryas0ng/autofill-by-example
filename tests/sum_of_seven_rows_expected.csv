11,42
15,74
2,64
5,68
3,72
2,72
4,74
43,72
5,34
6,31
9,31
3,23
4,28
2,30
5,29
2,45
6,46
1,40
8,39
6,31
1,25
21,24
3,3